.docker/postgres/sql-init/setup.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE db_test;
+
+GRANT ALL PRIVILEGES ON DATABASE db_test TO admin;
```

Comment:
my comment about whether we want to make the test user an env var would affect this file too 😄

Reply:
For this one is a little bit complicated, we would need to think in a way to inject those variables in an .sql file.